includes/class-subscription.php

```diff
@@ -113,7 +116,7 @@ public function create()
                 'return_url'  => $this->return_url,
                 'webhook'     => $this->webhook_url,
                 'features'    => $features,
-                'test'        => $this->is_test_subscription(),
+                'test'        => isset($this->test) ? $this->test : $this->is_test_subscription(),
```

Comment:
No hay error acá, pero`$this->test` siempre está definido porque es una propiedad en la línea 9. Se puede reescribir para evaluar si es true directamente:
```suggestion
                'test'        => $this->test ?: $this->is_test_subscription(),
```